less/assessment.less

```diff
@@ -0,0 +1,11 @@
+.no-marking {
+	.buttons-marking-icon {
+		display: none;
+	}
+	.question-component {
+		.icon-tick,
+		.icon-cross {
+			display: none!important;
```

Comment:
Just to note we can avoid the use of `!important` by amending to:
```
.no-marking {
	.component-inner .component-widget.disabled.submitted.complete .icon-tick,
	.component-inner .component-widget.disabled.submitted.complete .icon-cross,
	.buttons-marking-icon {
		display: none;
	}
}
```
Tested on GMCQ, Matching, MCQ, Slider and Text Input components.